spec/frame_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'frame'
+
+describe Frame do
+  let(:frame) { Frame.new }
+  describe '#rolls' do
+    it 'should go to the next roll' do
```

Comment:
I'm finding this it statement a little bit unclear: it looks like the test is testing whether a new frame starts out with no rolls in it, but the statement implies it should be testing if it goes to the next roll; maybe either the test could be changed or the it statement could be reworded?

Reply:
Good call, I was writing the test with the intention to test it going to the next one, and then decided it would be better practice to see if it is initialized as an empty array first, but forgot to change the it statement.